tests/integration/testdata/buildcmd/provided_create_new_file/Makefile

```diff
@@ -0,0 +1,6 @@
+build-Function:
```

Comment:
all the files from this new `provided_create_new_file` directory are a copy of the `Provided` directory, with the only difference being `touch file-created-from-make-command` in the makefile